coa/pkg/apis/v1alpha2/host/host.go

```diff
@@ -127,6 +135,21 @@ func (h *APIHost) Launch(config HostConfig,
 					}
 				}
 
+				if config.API.KeyLock.Provider.Type != "" {
+					if h.SharedKeyLockProvider == nil {
```

Comment:
Do we want to only initialize a global key lock provider in the APIHost? If user wrongly define a shared provider, looks like the CreateProvider will return the error and terminate the process. Shall we make the error message more clear?
err = errors.New("globalMemoryKeyLock should be init before useign shared mode")

 And if user wrongly define a dedicated provider, the provider cannot be used by managers.

Reply:
this provider here are not passed to the managers. if user wrongly define a dedicated provider here. a shared mode provider cannot be initialized.

Reply:
add global check in the host file